docs/source/package/tools.rst

```diff
@@ -2,7 +2,11 @@
 Additional Tools
 ================
 
+Sopel provides utilities for some commonly used plugin functionality. Most of
+these utility features have been collected into submodules for convenience:
+
 .. toctree::
+   :titlesonly:
```

Comment:
This one I think I agree with, but the rendered docs would be _much_ more helpful if the `On this page` bar listed every anchor reachable on the page.

Reply:
What anchor(s) is it missing? It doesn't include the top-level heading (page title), but looks like everything else is there. (The subpages are not "anchor[s] reachable on the page"; they are subpages.)

Reply:
Oh, you're right, I guess that's something I've been slightly misunderstanding. I know that the entire tree is in practice visible in the left side-bar, but I think it's kinda hard to drill down into those and that is probably influencing my remarks here.